app/src/main/java/com/example/stromtracker/ui/geraete/GeraeteListAdapter.kt

```diff
@@ -35,44 +36,58 @@ class GeraeteListAdapter(
 
 
 
-        override fun onBindViewHolder(holder: GeraeteListAdapter.GeraeteViewHolder, position: Int) {
-            holder.mTextView.text = geraeteList[position].getName()
-            //.withSign(1) lässt den "Verbrauch" bzw. die Produktion von Produzenten positiv anzeigen, da diese als negativer Verbrauch in der DB gespeichert ist
-            holder.mVerbrauchView.text =
-                geraeteList[position].getJahresverbrauch().withSign(1).toString()
```

Comment:
Das .withSign(1) brauche Ich dann noch, da die Produzenten den selben List Adapter verwenden und diese als negative Zahl in der DB gespeichert sind.
Das kann ich dann aber nochmal anpassen

Reply:
Wie gesagt, da in Develop der ListAdapter sowieso noch nicht für Produzenten angepasst wurde, und ich das Runden genau da hinzufügen musste, habe ich das gelöscht.

Du musst ja sowieso die if-Klause noch machen, da kannst du es dann hinzufügen.